ambrs/ppe.py

```diff
@@ -29,6 +29,8 @@ class EnsembleSpecification:
     flux: RVFrozenDistribution
     relative_humidity: RVFrozenDistribution
     temperature: RVFrozenDistribution
```

Comment:
In PartMC, one can prescribe a temperature profile. As I recall, MAM4 does not (yet) allow temperature to vary over time. This may be something we want to modify down the line, but not a high priority.